functional.spec.js

```diff
@@ -16,7 +16,7 @@ describe('functional', () => {
     }),
   );
 
-  afterAll((done) => server.close(done));
+  afterAll((done) => { server.close(done); });
```

Comment:
@stefanbuck does this look right to you?

Reply:
Yes, it does look correct. In the node.js version we use, `server.close` returns an instance of the server. The arrow function has a single expression, that will implicit return the return value of `server.close`.

Reply:
Ok cool, I'll fix this up and get it merged later tonight then.